Templates/Tests/Generated/AutoMockable.generated.swift

```diff
@@ -15,26 +15,44 @@ import AppKit
 
 
 
+
```

Comment:
Maybe we should try to avoid adding some empty lines here.
This comes from the empty lines between the `macro` declarations, maybe we could just remove them from the template?